scripts/main/photo.js

```diff
@@ -1021,6 +1021,46 @@ photo.getArchive = function (photoIDs, kind = null) {
 	location.href = "api/Photo::getArchive" + lychee.html`?photoIDs=${photoIDs.join()}&kind=${kind}`;
 };
 
+photo.qrCode = function (photoIDs) {
```

Comment:
I know, you just adapted the style which has been there. But there is a rather large pending PR which introduces jsDoc comments and type annotations for all methods. These are crucial to allow static code analyses and improve code quality. For new code I would like to ask for properly annotated methods.

```suggestion
/**
 * Shows a dialog to share the download URL via a QR code.
 *
 * @param {string[]} photoIDs
 * @returns {void}
 */
photo.qrCode = function (photoIDs) {
```

Reply:
https://github.com/LycheeOrg/Lychee-front/pull/287/commits/55fe0cb2375ba5d9c5b25f5db178c189a4e5b6bc